src/main/java/com/amazonaws/ant/cloudformation/DescribeStackTask.java

```diff
@@ -0,0 +1,145 @@
+package com.amazonaws.ant.cloudformation;
```

Comment:
Go ahead and throw in the standard Apache 2.0 header comment that we use in other Java source files in this project.